### DISCUSSION TITLE
带缩略轴柱状图有类似maxValueSpan的配置项吗

### DISCUSSION THREAD

**Discussion by sdmu-gaoqi** [antvis/G2Plot#2364] [Ideas]
* **G2Plot Version**:
* **Platform**:
* **Mini Showcase(like screenshots)**:
* **CodePen Link**:

**Comment by visiky** [comment]
这是啥？附着下 demo 示意

**Comment by sdmu-gaoqi** [comment]
是这样的
    slider: {
      start: 0.0,
      end: 0.3,
    }
我在配置柱状图的slider时疑惑slider组件是否有缩放的范围限制
例如目前slider的start可以从0.0至1.0,是否有API支持可以限制在拖拽拉伸时实现start与end的相差范围不能超出某个值

**Comment by hustcc** [comment]
> 是这样的
> slider: {
> start: 0.0,
> end: 0.3,
> }
> 我在配置柱状图的slider时疑惑slider组件是否有缩放的范围限制
> 例如目前slider的start可以从0.0至1.0,是否有API支持可以限制在拖拽拉伸时实现start与end的相差范围不能超出某个值

目前还没有！
